Assets/Character/AaronTestAssets/PlayerMovement.cs

```diff
@@ -23,6 +23,8 @@ public class PlayerMovement : MonoBehaviour {
     private SpriteRenderer sr;
```

Comment:
# of private vars seem excessive. doesn't seem like the entire player movement class always needs to know heightDifference, groundDistance, downRay, etc. Just something to be away of